flink-bash-java-utils/pom.xml

```diff
@@ -0,0 +1,108 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+	<modelVersion>4.0.0</modelVersion>
+
+	<parent>
+		<groupId>org.apache.flink</groupId>
+		<artifactId>flink-parent</artifactId>
+		<version>1.11-SNAPSHOT</version>
+		<relativePath>..</relativePath>
+	</parent>
+
+	<artifactId>flink-bash-java-utils_${scala.binary.version}</artifactId>
+	<name>flink-bash-java-utils</name>
+
+	<packaging>jar</packaging>
+
+	<dependencies>
+		<dependency>
+			<groupId>org.apache.logging.log4j</groupId>
+			<artifactId>log4j-slf4j-impl</artifactId>
+			<scope>compile</scope>
```

Comment:
What is in the build result `bash-java-utils.jar`? does it contain only `BashJavaUtils.class` and `log4j2.properties`? Everything else can come from other jars. I think compile scope will include the dependencies into the result.

Reply:
The build result includes not only `BashJavaUtils.class` and `log4j2.properties`, but also the log4j dependencies.

The log4j dependencies are included intentionally in FLINK-15519 by @dawidwys. IIUC, this is to make sure the version of log4j classes used by `BashJavaUtils` match the packaged `log4j2.properties`.

Currently, log4j dependencies are excluded from `flink-dist*.jar`, to allow users to choose different log4j versions and provide both jars and the properties file. For `BashJavaUtils`, we need to include the properties file to make sure logs are outputted to the console. Thus, we need to also include the dependencies to make sure they're compatible with the properties file.